index.html

```diff
@@ -1,222 +1,267 @@
-                            <li>Css</li>
-                        </ul>
-                    </ul>
-                    <ul>
-                        <li class="bold">
-                            <img src="images/Rectangle 56.png" alt="rectangle">
-                            Frameworks
-                        </li>
-                        <ul class="skills">
-                            <li>Bootstrap</li>
-                            <li>Ruby on Rails</li>
-                            <li>RSPec</li>
-                            <li>CapyBara</li>
-                            <li>Selenium</li>
-                        </ul>
+                </ul>
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 56.png" alt="rectangle">
+                        Frameworks
+                    </li>
+                    <ul class="skills">
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
                     </ul>
-                    <ul>
-                        <li class="bold">
-                            <img src="images/Rectangle 57.png" alt="circle">
-                            Skills
-                        </li>
-                        <ul class="skills">
-                            <li>Codekit</li>
-                            <li>Githubt</li>
-                            <li>Codepen</li>
-                            <li>Gitlab</li>
-                            <li>Terminal</li>
-                        </ul>
+                </ul>
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 57.png" alt="circle">
+                        Skills
+                    </li>
+                    <ul class="skills">
+                        <li>Codekit</li>
+                        <li>Githubt</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
                     </ul>
+                </ul>
+            </div>
+        </section> <!-- end about me section -->
+
+        <!-- start contact form section -->
+        <section class="contact flex" id="contact">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </h3>
+            <form action="https://formspree.io/f/mayvdjkb" method="POST" class="form flex" id="contact-form">
+                <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" />
+                <div class="full-name flex">
+                    <input type="text" name="first_name" placeholder="First Name" maxlength="30" />
+                    <input type="text" name="last_name" placeholder="Last Name" maxlength="30" />
```

Comment:
- Hey, please, let's make this input field required to ensure the form is not submitted on an empty input field.